output/typescript/types.ts

```diff
@@ -7855,6 +7866,14 @@ export interface OpenJobResponse extends ResponseBase {
   opened: boolean
 }
 
+export interface OpenPointInTimeRequest extends RequestBase {
+  index: IndexName
```

Comment:
Should these be here? I believe the request for this is empty and the index and keep_alive come from the path and query?

Reply:
`keep_alive` is a query param and `index` is in the url path according to the docs: https://www.elastic.co/guide/en/elasticsearch/reference/master/point-in-time-api.html#point-in-time-api-example

Reply:
Sorry, didn't realise I was looking at the output!